plugins/modules/win_dns_record.ps1

```diff
@@ -16,7 +16,7 @@ $spec = @{
         value = @{ type = "list"; elements = "str"; default = @() ; aliases=@( 'values' )}
         weight = @{ type = "int"}
         zone = @{ type = "str"; required = $true }
-        computer_name = @{ type = "str" }
+        server = @{ type = "str" }
```

Comment:
Need to make sure we don't break any existing playbooks by accepting it as an alias

```suggestion
        server = @{ type = "str"; aliases = @("computer_name") }
```